source/jormungandr/jormungandr/utils.py

```diff
@@ -693,3 +693,105 @@ def can_connect_to_database():
         return False
 
     return True
+
+
+def create_journeys_request(origins, destinations, datetime, clockwise, journey_parameters, bike_in_pt):
+    req = request_pb2.Request()
+    req.requested_api = type_pb2.pt_planner
+    for stop_point_id, access_duration in origins.items():
+        location = req.journeys.origin.add()
+        location.place = stop_point_id
+        location.access_duration = access_duration
+
+    for stop_point_id, access_duration in destinations.items():
+        location = req.journeys.destination.add()
+        location.place = stop_point_id
+        location.access_duration = access_duration
+
+    req.journeys.night_bus_filter_max_factor = journey_parameters.night_bus_filter_max_factor
+    req.journeys.night_bus_filter_base_factor = journey_parameters.night_bus_filter_base_factor
+
+    req.journeys.datetimes.append(datetime)
+    req.journeys.clockwise = clockwise
+    req.journeys.realtime_level = realtime_level_to_pbf(journey_parameters.realtime_level)
+    req.journeys.max_duration = journey_parameters.max_duration
+    req.journeys.max_transfers = journey_parameters.max_transfers
+    req.journeys.wheelchair = journey_parameters.wheelchair
+    if journey_parameters.max_extra_second_pass:
+        req.journeys.max_extra_second_pass = journey_parameters.max_extra_second_pass
+
+    for uri in journey_parameters.forbidden_uris:
+        req.journeys.forbidden_uris.append(uri)
+
+    for id in journey_parameters.allowed_id:
+        req.journeys.allowed_id.append(id)
```

Comment:
I'm not sure it works :)
```suggestion
    req.journeys.forbidden_uris = [uri for uri in journey_parameters.forbidden_uris]

    req.journeys.allowed_id = [id for id in journey_parameters.allowed_id]
```

Reply:
I am not really sure either. Here we are building a protobuf object, and not a python dict